lib/consensus.go

```diff
@@ -9,9 +9,9 @@ type Consensus interface {
 	GetNode() *sebaknode.LocalNode
 	HasMessage(sebakcommon.Message) bool
 	HasMessageByHash(string) bool
-	ReceiveMessage(sebakcommon.Message) (Ballot, error)
-	ReceiveBallot(Ballot) (VotingStateStaging, error)
+	ReceiveMessage(sebakcommon.Message) error
+	//ReceiveBallot(Ballot) (VotingStateStaging, error)
 
-	AddBallot(Ballot) error
-	CloseConsensus(Ballot) error
+	//AddBallot(Ballot) error
+	//CloseConsensus(Ballot) error
```

Comment:
These methods(GetNode, HasMessage, HasMessageByHash, ReceiveMessage) are not used.
@spikeekips How about removing consensus.go or redesign interface? :)

Reply:
Don't comment out code, remove it instead, please. That's what VCS are for.

Reply:
@Charleslee522 Yes, I agree. We can remove `Consensus` interface.